alvr/dashboard/src/dashboard/components/setup_wizard.rs

```diff
@@ -114,12 +118,38 @@ Make sure you have at least one output audio device.",
                 ui,
                 "Software requirements",
                 r"To stream the Quest microphone on Windows you need to install VB-Cable or Voicemeeter.
-On Linux some feaures are not working and should be disabled (foveated encoding and color correction) and some need a proper environment setup to have them working (game audio and microphone streaming).",
+On Linux, game audio and microphone might require pipewire and On connect/On disconnect script.",
                 |ui| {
                     if ui.button("Download VB-Cable").clicked() {
                         ui.ctx()
                             .open_url(OpenUrl::same_tab("https://vb-audio.com/Cable/"));
                     }
+                    if ui
+                        .button("'On connect/On disconnect' audio script")
+                        .clicked()
+                    {
+                        ui.ctx()
+                            .open_url(OpenUrl::same_tab("https://github.com/alvr-org/ALVR-Distrobox-Linux-Guide/blob/main/audio-setup.sh"));
```

Comment:
Actually, this button feels a bit misleading. I feel like it should set the on connect/disconnect script instead of just linking to the code.

Reply:
for that it needs to download it and place somewhere
i thought about that, but was unsure if to proceed or not

Reply:
Definitely it would be a QoL improvement

Reply:
Added download to config folder, set in settings, using reqwest::blocking. It does block ui when it downloads, but i couldn't find a way to *not* do it without using async code. It's very little download and it does that only in setup screen, seems to be fine to leave it as is?